test/util/model-tests.js

```diff
@@ -235,6 +235,19 @@ module.exports = function defineModelTestsWithDataSource(options) {
       });
     });
 
+    describe('Model.exists(id, [callback])', function() {
+      it('Check existing of a model instance in the attached data source', function(done) {
```

Comment:
Test names should read like English sentences. This one does not:

```
Model.exists(id, [callback]) Check existing of a model instance in the attached data source
```

To speed up the process, I'll edit the test name myself and force-push into your feature branch.